apps/communications/dialer/style/call_log.css

```diff
@@ -175,7 +175,13 @@ ol, ul {
 }
 
 .log-item .primary-info-main {
+  min-width: 0;
```

Comment:
We need this because of flex parent, see https://bugzilla.mozilla.org/show_bug.cgi?id=1043520 for deta for more details.